test/tcp/TcpClient.cpp

```diff
@@ -27,6 +28,46 @@ class Callback : public AsyncSocket::ConnectCallback {
     std::cout << "TODO error" << ex.what() << " " << ex.getType() << "\n";
   }
 };
+
+class FollyKeepaliveTimer : public KeepaliveTimer {
```

Comment:
do you think we should put the FollyKeepaliveTimer under src/<something>? it will be commonly used..